[2025-10-28T23:45:01.157+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-28T23:44:00+00:00 [queued]>
[2025-10-28T23:45:01.166+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-28T23:44:00+00:00 [queued]>
[2025-10-28T23:45:01.167+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-28T23:45:01.181+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ConsumKafka> on 2025-10-28 23:44:00+00:00
[2025-10-28T23:45:01.194+0000] {standard_task_runner.py:60} INFO - Started process 235 to run task
[2025-10-28T23:45:01.198+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'ConsumKafka', 'scheduled__2025-10-28T23:44:00+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmpgsz4aq1y']
[2025-10-28T23:45:01.200+0000] {standard_task_runner.py:88} INFO - Job 74: Subtask ConsumKafka
[2025-10-28T23:45:01.267+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-28T23:44:00+00:00 [running]> on host 9680281a20bc
[2025-10-28T23:45:01.370+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='ConsumKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-28T23:44:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-28T23:44:00+00:00'
[2025-10-28T23:45:01.377+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:45:01.383+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.5', 9092)]>: Broker version identified as 2.6
[2025-10-28T23:45:01.384+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:45:01.385+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('source',)
[2025-10-28T23:45:01.385+0000] {dag1_kafka_processing.py:71} INFO - Consommateur Kafka créé pour le topic 'source'. En attente de messages...
[2025-10-28T23:45:01.389+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:45:01.390+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:45:01.390+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:45:01.494+0000] {cluster.py:401} INFO - Coordinator for group/***-consumers-dag1 is ('coordinator-1', 'kafka', 9092, None)
[2025-10-28T23:45:01.495+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group ***-consumers-dag1
[2025-10-28T23:45:01.495+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-10-28T23:45:01.496+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-28T23:45:01.497+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: NodeNotReadyError: coordinator-1
[2025-10-28T23:45:01.499+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:45:01.500+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:45:01.601+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-28T23:45:01.605+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-a3af5cbb-c898-4df8-8940-f105001a51f3 for group ***-consumers-dag1; will retry join-group
[2025-10-28T23:45:01.605+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: [Error 79] MemberIdRequiredError
[2025-10-28T23:45:01.606+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-28T23:45:04.612+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 6 (member_id: kafka-python-2.2.15-a3af5cbb-c898-4df8-8940-f105001a51f3, protocol: range)>
[2025-10-28T23:45:04.613+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-28T23:45:04.618+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('source', 0)]
[2025-10-28T23:45:04.619+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('source', 0),) for group ***-consumers-dag1
[2025-10-28T23:45:11.391+0000] {base.py:1052} INFO - Stopping heartbeat thread
[2025-10-28T23:45:11.392+0000] {base.py:891} INFO - Leaving consumer group (***-consumers-dag1).
[2025-10-28T23:45:11.397+0000] {base.py:906} INFO - LeaveGroup request for group ***-consumers-dag1 returned successfully
[2025-10-28T23:45:11.398+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:45:11.399+0000] {fetcher.py:786} INFO - Fetch to node 1 failed: Cancelled: <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>
[2025-10-28T23:45:11.399+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:45:11.400+0000] {dag1_kafka_processing.py:77} WARNING - Aucun message reçu du topic 'source'. Le DAG s'arrêtera.
[2025-10-28T23:45:11.400+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-28T23:45:11.413+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=ConsumKafka, execution_date=20251028T234400, start_date=20251028T234501, end_date=20251028T234511
[2025-10-28T23:45:11.463+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-28T23:45:11.498+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
